src/ontology/mondo-edit.obo

```diff
@@ -421008,8 +421012,9 @@ synonym: "Feline leukemias" RELATED [MESH:D016582]
 synonym: "leukemias, Feline" RELATED [MESH:D016582]
 xref: MESH:D016582 {source="MONDO:equivalentTo"}
 xref: UMLS:C0085164 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005059 {source="MESH:D016582"} ! leukemia
 is_a: MONDO:0024912 {source="MESH:D016582"} ! cat disease
+intersection_of: MONDO:0024912 ! cat disease
```

Comment:
I dont think you need that. It should be:

"disease" (MONDO:1) and
in taxon some cat and
has analog some leukemia

This is at least how I understood this?

Reply:
this makes the patterns much easier to manage.

Reply:
ah, got it. I wasn't sure, this makes sense, thanks.

Reply:
fixed